eNMS/controller/inventory.py

```diff
@@ -49,12 +49,15 @@ def web_connection(self, device_id, **kwargs):
             return {"alert": "Unauthorized authentication method."}
         device = db.fetch("device", id=device_id, rbac="connect")
         port, endpoint = self.get_ssh_port(), str(uuid4())
-        command = f"flask run -h 0.0.0.0 -p {port}".split()
+        command = f"python3 -m flask run -h 0.0.0.0 -p {port}"
         if self.settings["ssh"]["bypass_key_prompt"]:
             options = "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
         else:
             options = ""
         environment = {
+            **{key: str(value) for key, value in self.settings["ssh"]["web"].items()},
+            "LC_ALL": "en_US.utf-8",
```

Comment:
redundant with what you put in settings.json ?

Reply:
yes, removed